R/mail.R

```diff
@@ -6,110 +6,249 @@
 
 ############## Downloader ##############
 
-#' Download all pipermail files in an archive
-#' @param url An url pointing to a pipermail archive
-#' @return Returns `destination`, a vector of the downloaded files in the current working directory
+#' Download all pipermail files in an archive as mbox files
+#' @param mailing_list The name of the mailing list being downloaded
```

Comment:
Functions can have descriptions (and they can be fairly long). I would like you to add to the description of the function that it will download a .html file containing the actual URLs for the files, and subsequently download the files in .txt or .gz to convert to .mbox here. See other functions in git.R such as parse_gitlog() for examples.